torch/custom_class.h

```diff
@@ -153,6 +153,26 @@ class class_ {
     return *this;
   }
 
+  template <typename GetterFunc, typename SetterFunc>
+  class_& def_property(
+      std::string name,
+      GetterFunc getterFunc,
+      SetterFunc setterFunc,
+      std::string getter_doc_string = "",
+      std::string setter_doc_string = "") {
```

Comment:
I think the user experience would be better with only one docstring (for the property) instead of two.

Reply:
sounds good